hibernate-core/src/main/java/org/hibernate/dialect/cleaner/DatabaseCleaner.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect.cleaner;
+
+import java.sql.Connection;
+
+/**
+ * @author Christian Beikov
+ */
+public interface DatabaseCleaner {
+
+	static void clearSchemas() {
+		final DatabaseCleaner cleaner = DatabaseCleanerContext.CLEANER;
+		if ( cleaner != null ) {
+			JdbcConnectionContext.work( cleaner::clearAllSchemas );
+		}
+	}
+
+	static void clearData() {
+		final DatabaseCleaner cleaner = DatabaseCleanerContext.CLEANER;
+		if ( cleaner != null ) {
+			JdbcConnectionContext.work( cleaner::clearAllData );
+		}
+	}
```

Comment:
Dunno where these two methods are used, but they shouldn't be part of the API since they are using `JdbcConnectionContext`